app/pocket/rpc/client.gen.go

```diff
@@ -0,0 +1,444 @@
+// Package rpc provides primitives to interact with the openapi HTTP API.
```

Comment:
I thought about it but I wanted to show the generated files and I didn't want to mix this with the protogen since they have separate inputs.

- I have git-ignored the generated files
- I have updated the `Makefile` `develop_test`
- I have refactored the commands necessary to setup development into `develop_start`
- I realized that calling mockgen before protogen_local created some issues, I guess because some mocks depend on generated protos, sorted accordingly by generating protos first and tested locally
- I have updated the various calls (docker files, GitHub workflow)
- I have updated `docs/development/README.md`

Addressed in https://github.com/pokt-network/pocket/pull/177/commits/653f6d702592f0a9ee6cb1903b60a1c9536ff583

Reply:
Perfect 👌

Reply:
I think we need to do a `git rm` for the gitignore to apply.